imports/blocks/give/fieldsets/shared/Icon.js

```diff
@@ -0,0 +1,12 @@
+import { PropTypes } from "react";
+import AccountType from "../../../../components/accountType";
+
+const Icon = ({ cardType }) => (
```

Comment:
There were a lot of places where `<AccountType width="30px" height="21px"` was being used. So my first notion was to abstract it out. I did end up simplifying some of the logic surrounding rendering Icons, which makes the extra component not as necessary. I'm happy to remove it, but I'm not mad at it this way either.